### DISCUSSION TITLE
Use icons locally as static assets

### DISCUSSION THREAD

**Discussion by mrcasablr** [fomantic/Fomantic-UI#444] [Elements (button, image, label, loader, ...)] [answered]
# Help Wanted
I'm trying to run fomantic-ui with semantic-ui-vue and so far everything's working great. Fomantic CSS is downloaded from a CDN. I would like to run this locally.

## Problem
Currently CSS is loaded via -
```
<link
      rel="stylesheet"
      href="https://cdn.jsdelivr.net/npm/fomantic-ui@2.7.1/dist/semantic.min.css"
    />
```

On loading the css locally,  I see icons replaced with empty square and see the following errors on browser console -
```
Failed to decode downloaded font: http://localhost:8080/assets/css/themes/default/assets/fonts/icons.woff2
Failed to decode downloaded font: http://localhost:8080/assets/themes/default/assets/fonts/icons.woff
Failed to decode downloaded font: http://localhost:8080/assets/themes/default/assets/fonts/icons.ttf
```
I do not have these files locally. Where do I get them?

**Comment by y0hami** [comment, marked as answer]
@mrcasablr If your using NPM you can install [`fomantic-ui-css`](https://www.npmjs.com/package/fomantic-ui-css) and it will download the fonts and you can then import the CSS file and it should just work. If you aren't using NPM I would recommend cloning the CSS repository which comes with the font files. You could also download the fonts files from the repo itself https://github.com/fomantic/Fomantic-UI-CSS

**Comment by mrcasablr** [comment]
Thank you! This works and convenient!
